File: 1-Basics/1.2-Basic_Maths/solution_in_cpp/3-Reverse_Number.cpp
#include<bits/stdc++.h>
using namespace std;


int Reverse_number(int n){
    int num = 0;
    int rem = n;
    while(rem != 0){
        int last = rem % 10;
        num = (num*10) + last;
        rem = rem/10;
    }

    return num;
}

int main(){
    cout<<"Enter Number N : ";
    int n;
    cin>>n;

    cout<<Reverse_number(n);

    return 0;
}

File: 1-Basics/1.2-Basic_Maths/solution_in_python/3-Reverse_Number.py
def Reverse_number(n):
    sign = 1 if n>0 else -1
    num = 0
    rem = abs(n)
    while(rem != 0):
        last = rem%10
        num = (num*10) + last
        rem = rem//10
    
    return sign*num


n = int(input("Enter NUmber N : "))

print(Reverse_number(n))